apps/src/templates/projects/SectionProjectsListWithData.jsx

```diff
@@ -0,0 +1,51 @@
+import React, {Component, PropTypes} from 'react';
+import Spinner from '@cdo/apps/code-studio/pd/components/spinner';
+import SectionProjectsList from './SectionProjectsList';
+
+class SectionProjectsListWithData extends Component {
+  static propTypes = {
+    sectionId: PropTypes.string,
+    studioUrlPrefix: PropTypes.string
+  };
+
+  state = {
+    projectsData: {},
+    isLoading: true
+  };
+
+  componentDidMount() {
+    const projectsDataUrl = `/dashboardapi/v1/projects/section/${this.props.sectionId}`;
+    $.ajax({
```

Comment:
same thoughts about eventually moving this to redux as [this comment](https://github.com/code-dot-org/code-dot-org/pull/26401#discussion_r242999301)